sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -322,7 +320,9 @@ export class KeyVaultAccessControlClient {
   ): PagedAsyncIterableIterator<KeyVaultRoleAssignment> {
     const span = createSpan("listRoleAssignments", options);
     const updatedOptions: ListRoleAssignmentsOptions = {
-      ...options,
+      abortSignal: options.abortSignal,
```

Comment:
does this removal of spread do anything useful? It seems like if we expanded `OperationOptions` tomorrow, it would be problematic, since it would silently drop the new options

Reply:
You're right! I'll undo this